.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+}
```

Comment:
Shouldn't commit this file IMO. Remove it and do a rebase to remove it from history?